knowledge-base/tutorials.md

```diff
@@ -0,0 +1,26 @@
+I think this is outdated and with the github.io we do not need it
```

Comment:
I think it is still valid. We will keep those tutorials up, the documentation is a technical addition to the tutorials, which are more a step by step guide to achieve a goal.

Reply:
Yes, I meant that we reference the tutorials on the github.io https://sap.github.io/cloud-sdk/docs/overview/getting-started
So I wonder if it makes sense to keep a markdown in the reop?

Reply:
Ah no, you can delete it, good point.